packages/now-cli/test/integration.js

```diff
@@ -2238,25 +2065,8 @@ test('fail `now dev` dev script without now.json', async t => {
   );
 });
 
-test('print correct link in legacy warning', async t => {
-  const deploymentPath = fixture('v1-warning-link');
-  const { exitCode, stderr, stdout } = await execute([
-    deploymentPath,
-    '--confirm',
-  ]);
-
-  console.log(stderr);
-  console.log(stdout);
-  console.log(exitCode);
-
-  // It is expected to fail,
-  // since the package.json does not have a start script
-  t.is(exitCode, 1);
-  t.regex(stderr, /migrate-to-vercel/);
-});
-
 test('`vercel rm` removes a deployment', async t => {
-  const directory = fixture('builds');
+  const directory = fixture('now-dev-fail-dev-script');
```

Comment:
I don't think you can use `now-dev-fail-dev-script` because it didn't create a deployment so there is nothing to delete. Let's try a previous test we expect to be deployed properly.

```suggestion
  const directory = fixture('static-deployment');
```